tests/HasReviewRatingTest.php

```diff
@@ -473,7 +473,7 @@ public function it_can_get_average_rating()
         ]);
         $this->assertTrue($this->testModel->hasReview());
         $this->assertEquals(3, $this->testModel->numberOfRatings());
-        $this->assertEquals(4.66666666666667, $this->testModel->averageRating());
+        $this->assertEquals(4.667, $this->testModel->averageRating(3));
```

Comment:
The tests before are failing here because it's showing a different `avg` result on php74 and php81 that's why I only used 3 decimal places.